### PR TITLE
chore(cloudformation-diff): Changed to a more abstract type.

### PR DESCRIPTION
**Pull request by taichi**
Stream that formatSecurityChanges accepts as its first argument, because it does not require a tight coupling with TTY.
This declaration is probably a typo.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Review by TheRealAmazonKendra — CHANGES_REQUESTED** [review]
Can you please elaborate more on the problem you're trying to solve here? Is there an open issue for this? If not, I'm not sure I'm understanding why this change is needed.

**Comment by taichi** [comment]
In the current code, since it accepts WriteStream, attempting to compile with formatSecurityChanges in code intended to operate only in memory, such as in a browser, results in an error. I would like to run the code that uses this function in an environment without a TTY.

However, upon examining the contents of formatSecurityChanges, the processing itself does not have any dependencies on TTY and is written as portable code that can operate in any environment.

In other words, the author of this code likely intended to require a WriteableStream as an argument, rather than a WriteStream. These are two classes with very similar names, so I believe it may simply be a typo.

**Review by TheRealAmazonKendra — APPROVED** [review]

**Review by TheRealAmazonKendra — APPROVED** [review]
